### PR TITLE
Add a supported chain id to the env

### PR DESCRIPTION
**Pull request by georgeweiler**

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -16,3 +16,5 @@ cypress/plugins
 # Yarn
 node_modules/
 yarn-error.log
+
+.env
```
Comment (r-czajkowski) [inline .gitignore:20]: Can we add the `.env` file with default values for local development?

```
REACT_APP_SUPPORTED_CHAIN_ID=1337
REACT_APP_RPC_URL=http://localhost:8545
REACT_APP_WS_URL=ws://localhost:8545
```

Comment (r-czajkowski) [inline .gitignore:20]: I think we can add `.env.production` file as well which will be used during the build process and the file might look like:

`.env.production`
```
REACT_APP_SUPPORTED_CHAIN_ID=$chain_id
REACT_APP_RPC_URL=$rpc_url
REACT_APP_WS_URL=$ws_url
```

where environment variables(`chain_id`, `rpc_url`, `ws_url`) will be set in CI side.

File: src/utils/isSupportedNetwork.ts [hunk 2]
```diff
@@ -1,9 +1,19 @@
-import { ChainID } from "../enums"
+import { getEnvVariable } from "./getEnvVariable"
+import { EnvVariable } from "../enums"
+import * as net from "net"
```
Comment (r-czajkowski) [inline src/utils/isSupportedNetwork.ts:3]: I think it is unnecessary import.

**Review by r-czajkowski — CHANGES_REQUESTED** [review]

File: src/web3/connectors/walletConnect.ts [hunk 3]
```diff
@@ -1,7 +1,18 @@
...
-import { RPC_URL } from "../../config"
+import { getEnvVariable } from "../../utils/getEnvVariable"
+import { EnvVariable } from "../../enums"
+
+const rpcUrl = getEnvVariable(EnvVariable.RpcUrl)
+const supportedChainId = getEnvVariable(EnvVariable.SupportedChainId)
+
+console.log({
+  [Number(supportedChainId)]: rpcUrl as string,
+})
```
Comment (r-czajkowski) [inline src/web3/connectors/walletConnect.ts:10]: Do we need this?

**Review by r-czajkowski — APPROVED** [review]
LGTM!

**Comment by nkuba** [comment]
@georgeweiler having a PR description provided would be super helpful.

File: src/web3/connectors/walletConnect.ts (hunk 3 above)
Comment (r-czajkowski) [reply src/web3/connectors/walletConnect.ts]: Addressed in https://github.com/threshold-network/token-dashboard/pull/30/commits/2dcbcea3fabbb2e8818d9b6ada43dca2c1b83ad7

**Review by r-czajkowski — APPROVED** [review]
LGTM! Ropsten preview looks good and I was able to test the migration flow so let's :shipit:

**Pull request merged**